lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -1619,8 +1577,7 @@ EmitInfo WasmBytecodeGenerator::EmitExtractLaneExpr(Js::OpCodeAsmJs op, const Wa
     WasmTypes::WasmType resultType = signature[0];
     WasmTypes::WasmType simdArgType = signature[1];
 
-    EmitInfo simdArgInfo = PopEvalStack(simdArgType, _u("Argument should be of type M128"));
-    ReleaseLocation(&simdArgInfo);
```

Comment:
we should still release this -- but may need to reorder so that we release indexInfo, release simdArgInfo, and then acquire resultReg

Reply:
Good catch, thank you. I am not sure how I lost this line, it seems to be working.